### ISSUE TITLE
What's the content of val.pkl in train.py?

### ISSUE THREAD

**Issue by victorywys** [Derpimort/VGGVox-PyTorch#1]
Hi,
    When I'm trying to train the model with your code, I found there's a file named "val.pkl" needed as the validation set(if I understand correctly). So how can I generate this file given the official dataset? Or am I missing something of the dataset to run the experiment?

**Comment by Derpimort** [comment]
Thanks for pointing that out. I have uploaded the file in 5e22cd2. That file just contains the lengths of pre-processed segments, i needed it for batch-wise processing of validation data. I'll upload the code to generate the file if needed.
